### DISCUSSION TITLE
Custom Dirs in Nuxt3?

### DISCUSSION THREAD

**Discussion by lihe07** [nuxt/nuxt#20145] [Questions] [answered]
In Nuxt 2, path of built-in dirs can be configured by `dir` option.
However, when I switched to Nuxt 3, I figured out that this configuration no longer working:
```js
dir: {
    assets: '../../assets',
}
```
I'm using monorepo and I hope all nuxt apps can share a single assets folder.

**Comment by lihe07** [comment]
Temporarily resolved using symbol links. HMR seems to work fine. Are there any better solutions?

**Comment by ivodolenc** [comment, marked as answer]
Try this (from the [docs](https://nuxt.com/docs/api/configuration/nuxt-config#assets)):

```js
// nuxt.config.ts

export default defineNuxtConfig({
  dir: {
    assets: '../your-dir-location'
  },
})
```

**Comment by maou-shonen** [comment]
Although this is not a direct answer to your question, generally speaking, layers are the best choice for a monorepo.
https://nuxt.com/docs/getting-started/layers
